### DISCUSSION TITLE
How to access secured image with Authorization header

### DISCUSSION THREAD

**Discussion by nbarrera** [ant-design/ant-design#38969] [Q&A]
Hello,

I had some `<Avatar />` `<Image />` antd components using the `src=` attribute with an URL that included a `?token=` parameter, so in that way I was accessing the secured resource.

Now, the backend is requiring that the token goes on HTTP Authorization header, it can't be sent as an url parameter anymore.

I 've seen that the `<Upload />` component has support for that, but I haven't seen that same support for components retrieving resources like the ones I 've mentioned `Avatar`, `Image`

is there anyway to be able to get those resources (images, etc.) with tailored http headers? without sendig the access token as an url parameter?

thank you

**Comment by xXAvoraXx** [comment]
+++

**Comment by gergimenezp** [comment]
I am facing the same issue...
